orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationClient.java

```diff
@@ -111,6 +113,32 @@ public OrchestrationChatResponse chatCompletion(
     return new OrchestrationChatResponse(executeRequest(request));
   }
 
+  /**
+   * Generate a completion for the given prompt.
+   *
+   * @param prompt a text message.
+   * @return A stream of message deltas
+   * @throws OrchestrationClientException if the request fails or if the finish reason is
+   *     content_filter
+   */
+  @Nonnull
+  public Stream<String> streamChatCompletion(
```

Comment:
(Comment)

New method names match the existing `OpenAiClient` counter-part in `openai` module.